PowerUp/app/src/main/AndroidManifest.xml

```diff
@@ -141,14 +141,6 @@
             android:theme="@style/Theme.AppCompat.Light.NoActionBar" />
 
         <activity
-            android:name=".kill_the_virus_game.KillTheVirusTutorials"
-            android:configChanges="orientation|keyboardHidden"
-            android:label="@string/app_name"
-            android:screenOrientation="landscape"
-            android:theme="@style/Theme.AppCompat.Light.NoActionBar" />
-
-        <activity
-            android:name=".kill_the_virus_game.KillTheVirusGame"
             android:configChanges="orientation|keyboardHidden"
```

Comment:
There's a problem here. You have removed the android:name field from the <activity> tag.